package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "opentelemetry-base",
+  "version": "0.0.1",
+  "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "scripts": {
+    "fix": "lerna run fix",
+    "postinstall": "npm run bootstrap",
```

Comment:
Why was `npm` used over `yarn`? I try to go back to `npm` pretty much every year but in the end `yarn` is always superior.

Reply:
done